lib/releaselib.sh

```diff
@@ -116,10 +116,22 @@ release::set_build_version () {
   local main_job="${all_jobs[0]}"
 
   # Loop through the remainder, excluding anything specified by --exclude_suites
-  for ((i=1;i<=${#all_jobs[*]};i++)); do
-    re="\\b${all_jobs[$i]}\\b"
-    [[ ${exclude_suites[*]} =~ $re ]] || secondary_jobs+=(${all_jobs[$i]})
+  for ((i=1;i<${#all_jobs[*]};i++)); do
+    match="false"
```

Comment:
Thanks for doing this!

Everything below here can be simplified to:
```
[[ -n $exclude_patterns && ${all_jobs[$i]} =~ $exclude_patterns ]] \
     || secondary_jobs+=(${all_jobs[$i]})
```

And then just define exclude_patterns just below exclude_suites above:
```
local -a exclude_suites=($3)
# below construct joins array
local exclude_patterns=$(IFS="|";echo "${exclude_suites[*]}")
```